src/api/libcellml/analyserequation.h

```diff
@@ -42,7 +42,8 @@ class LIBCELLML_EXPORT AnalyserEquation
     {
         TRUE_CONSTANT, /**< An equation that computes a true constant, e.g. x = 3. */
         VARIABLE_BASED_CONSTANT, /**< An equation that computes a variable-based constant, e.g. x = y+z where y and z are true constants. */
-        RATE, /**< An equation that computes a rate, e.g. d(y)/dt = f(t, x). */
+        ODE, /**< An equation that computes an ODE, e.g. d(y)/dt = f(t, x). */
+        NLA, /**< An equation that computes one or several algebraic variables, e.g. x+y = f(z) where x and y are known, but not z.. */
```

Comment:
doc suggestions (similar update needed in the enum documentation above).

`ODE`: An equation that describes a differential equation, e.g., d(y)/d(t) = f(t, x)
`NLA`: An equation that can be solved to compute one or several algebraic variables, e.g. x+y = f(z) where x and y are known, but not z..

Reply:
`EXTERNAL`: A placeholder equation to capture the dependency of the model on an external variable.

Reply:
e38da60b